contracts/FIND.cdc

```diff
@@ -541,8 +545,12 @@ pub contract FIND {
 
 		//anybody should be able to fulfill an auction as long as it is done
 		pub fun fulfillAuction(_ name: String) 
-		pub fun buyAddon(name:String, addon: String, vault: @FUSD.Vault) 
-		access(contract) fun getNames() : [String]
+		// pub fun buyAddon(name:String, addon: String, vault: @FUSD.Vault) 
```

Comment:
why is buy Addon removed here?

Reply:
There should be no harm exposing buyAddon in the interface. 
Do we need this here? Will there be situations that find will help buy the lease add on for the user? 
But maybe we can leave this here just in case